Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -158,7 +158,7 @@ private void Update()
             // Try to manually poll the eye tracking data
             if (eyeGazeProvider != null 
                 && eyeGazeProvider.UseEyeTracking 
```

Comment:
I find the difference between "UseEyeTracking" and "IsGazeInputEyeBased" confusing. Shouldn't "UseEyeTracking" be used as an indicator to whether eye-gaze input is used in contrast to "IsGazeInputEyeBased"?

Reply:
Agree that the names are confusing, although having useEyeTracking does not guarantee that eye is input for gaze pointer.

Reply:
IsGazeInputEyeBased is a combination of UseEyeTracking which is the user enabled setting for having eye tracking as input method and IsEyeTrackingAvailable (which i want to rename to IsEyeTrackingDataValid) which is the flag for indicating if we did receive / could process eye tracking data from the hardware.

So basically IsGazeInputEyeBased is like a IsEyeTrackingEnabledAndValid flag (maybe we should rename it to that?)

Reply:
Its more clear to me too